packages/e2e/test/wallet_epoch_3/PersonalWallet/conwayTransactions.test.ts

```diff
@@ -135,7 +135,7 @@ export const paramsUpdate: Cardano.ProtocolParametersUpdateConway = {
   treasuryExpansion: '0.25'
 };
 
-describe('PersonalWallet/conwayTransactions', () => {
+describe.skip('PersonalWallet/conwayTransactions', () => {
```

Comment:
I am not happy to ship test skipping with this PR, but It will make the day easier ^^

Reply:
Neither me 😞 
but it would seem to be the lesser of two possible evils.

Reply:
Yep, considering it's being fixed [here](https://github.com/input-output-hk/cardano-js-sdk/pull/1414)